wgpu-core/build.rs

```diff
@@ -0,0 +1,15 @@
+fn main() {
+    // Setup cfg aliases
+    cfg_aliases::cfg_aliases! {
+        // Vendors/systems
+        apple: { any(target_os = "ios", target_os = "macos") },
+        linux: { all(unix, not(apple)) },
```

Comment:
do we need this one? appears to only be used in the `vulkan` define, so we can omit it here

Reply:
We don't really, I wasn't sure if it would help indicate the intent a little bit more, but if you prefer to omit it that's fine by me.